_pages/activities.html

```diff
@@ -121,7 +121,7 @@ <h3 id="activities">Activities</h3>
 
     <div class="row activity">
       <div id="addons-activity" class="col-md-8">
-        <p><b><a href="https://mozillacampusclubs.github.io/Build-a-Virtual-World/">Introduction to browser extensions</a></b></p>
+        <p><b><a href="https://developer.mozilla.org/en-US/Add-ons/WebExtensions">Introduction to browser extensions</a></b></p>
```

Comment:
Hey can you please update the link with the activity's link as you did below?

Reply:
done